packages/filebrowser/src/listing.ts

```diff
@@ -2694,6 +2700,11 @@ export namespace DirListing {
      * The default is `true`.
      */
     allowDragDropUpload?: boolean;
+
+    /**
+     * Callback for when a file is being open by the file browser
```

Comment:
I would not have guessed what it does by reading the description alone.

```suggestion
     * Callback overriding action performed when user asks to open a file.
     * The default is to open the file in the main area if it is not open already, or to reveal it otherwise.
```